README.md

```diff
@@ -49,7 +49,7 @@ confusing it for the typical _project analysis token_.
   if: always() && steps.sonarqube-analysis.outcome == 'success'
   with:
     tool: sonar
-    sonar-api-url: ${{ vars.SONAR_HOST_URL }}/api
+    sonar-host: ${{ vars.SONAR_HOST_URL }}/api
```

Comment:
I think we should remove the /api and add it inside the action logic, so teh user only have to provide the host url